generators/app/index.js

```diff
@@ -45,6 +45,10 @@ module.exports = yeoman.Base.extend({
       if (!this.mavenVersion) {
         throw new Error('We are unable to find a maven executable. A compatible version of maven is required.');
       }
+      try { var configJSON = this.fs.readJSON('.yo-rc.json') } catch (err) { /* ignore */ }
```

Comment:
I think we should initialize before the try; even though it's not strictly required for javascript.